app/src/main/java/com/ghomebyrw/gworker/activities/LoginActivity.java

```diff
@@ -33,42 +34,38 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_login);
+        ButterKnife.bind(this);
         getSupportActionBar().hide();
-
         jobClient = new JobClient();
-        etEmail = (EditText) findViewById(R.id.etEmailAddress);
-        etPassword = (EditText) findViewById(R.id.etPassword);
-        loginButton = (Button) findViewById(R.id.btnLogin);
-        loginButton.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                String emailAddress = etEmail.getText().toString();
-                String password = etPassword.getText().toString();
-                final LoginInfo logInInfo = new LoginInfo(emailAddress, password);
-                if (emailAddress.isEmpty() || password.isEmpty()) {
-                    Toast.makeText(LoginActivity.this, getString(R.string.email_password_not_null),
-                            Toast.LENGTH_SHORT).show();
-                } else {
-                    jobClient.login(logInInfo, new Callback<Boolean>() {
-                        @Override
-                        public void onResponse(Response<Boolean> response, Retrofit retrofit) {
-                            Log.i(LOG_TAG, "Login successful");
-                            Intent intent = new Intent(LoginActivity.this, MainActivity.class);
-                            intent.putExtra("logInInfo", logInInfo);
-                            startActivity(intent);
-                        }
+    }
 
-                        @Override
-                        public void onFailure(Throwable t) {
-                            Log.e(LOG_TAG, "Login information invalid");
-                            Toast.makeText(LoginActivity.this, getString(R.string.email_password_invalid),
-                                    Toast.LENGTH_SHORT).show();
-                        }
-                    });
+    @OnClick(R.id.btnLogin)
+    void logIn() {
```

Comment:
I tried that, but Butter Knife complained at compile time: `@OnClick methods must not be private or static.`